_gsocblogs/2022/blog_Acts_TirthankarMazumder.md

```diff
@@ -0,0 +1,24 @@
+---
+project: Acts 
+title: Vectorized Linear Algebra Implementation for Acts
+author: Tirthankar Mazumder
+[//]: # (photo: blog_authors/FirstLast.jpg)
```

Comment:
If you do not want to add a photo of yourself, we encourage people just add their link to their public GitHub avatar, as advertised in [this](https://groups.google.com/g/cern-hsf-gsoc-students-2022/c/w_fIfhDdcG8/m/Av5ceK0VCAAJ) message.

Reply:
Done.